app/javascript/chat/__tests__/__snapshots__/channelDetails.test.jsx.snap

```diff
@@ -195,7 +195,7 @@ preact-render-spy (1 nodes)
          
         <button
           type="button"
-          onClick={[Function value]}
+          onClick={[Function anonymous]}
```

Comment:
I'll be honest, I do not know why this changed to anonymous here and below in a bunch of other snapshot tests.

Reply:
My guess is the newer version of babel 🤷‍♂ . Happened to me too every single time I tried to upgrade Webpacker.

Reply:
Same. 🙃